src/openforms/authentication/contrib/digid/plugin.py

```diff
@@ -34,11 +38,21 @@ def start_login(
             "authentication:return",
             kwargs={"slug": form.slug, "plugin_id": "digid"},
         )
+
         # The return_url becomes the DigiD relay state - this is why we add the co-sign
         # param to that URL and not the `form_url`.
+        loa = form.authentication_backend_options.get(self.identifier, {}).get("loa")
+        signed_object = Signer().sign_object(
+            {
+                "loa": loa,
+                "next": form_url,  # bind loa to requested form
+                "utc": int(datetime.utcnow().timestamp()),  # bind loa to requested time
+            }
+        )
```

Comment:
Discussed in the office:

This could probably be simplified, since in this plugin will redirect to 
```
https://open-forms.nl/digid/login/?next=/auth/aanvraag-formulier-ooievaarspas/digid/return?next=https://open-forms..nl/aanvraag-formulier-ooievaarspas/startpagina?_start=1
```
And from the first next we can get the slug of the form and get from the database what the LoA should be.